tests/ert_tests/cli/test_model_factory.py

```diff
@@ -66,7 +66,7 @@ def test_init_iteration_number(self):
             ert = work_area.getErt()
             args = Namespace(iter_num=10, realizations=None)
             model = model_factory._setup_ensemble_experiment(
-                ert, args, ert.getEnsembleSize()
+                ert, args, ert.getEnsembleSize(), "0"
```

Comment:
As it might be a bit unclear what about using "experiment_id" or "exp_id" instead of "0"?

Reply:
Sure, fixed!